server/routes/createInvite.ts

```diff
@@ -13,43 +23,53 @@ const createInvite = async (models, req: UserRequest, res: Response, next: NextF
   const [chain, community] = await lookupCommunityIsVisibleToUser(models, req.body, req.user, next);
   if (!req.user) return next(new Error('Not logged in'));
 
-  if (!req.body.invitedAddress && !req.body.invitedEmail) return next(new Error('Must invite an address or email'));
-  if (req.body.invitedAddress && req.body.invitedEmail) return next(new Error('Can only invite either an address or email, not both'));
-  const chainOrCommObj = chain ? { chain_id: chain.id } : { offchain_community_id: community.id };
+  if (!req.body.invitedAddress && !req.body.invitedEmail) {
+    return next(new Error(Errors.NoEmailOrAddress));
+  }
+
+  if (req.body.invitedAddress && req.body.invitedEmail) {
+    return next(new Error(Errors.NoEmailAndAddress));
+  }
+
+  const chainOrCommObj = chain
+    ? { chain_id: chain.id }
+    : { offchain_community_id: community.id };
 
   // check that invitesEnabled === true, or the user is an admin or mod
   if (!community.invitesEnabled) {
-    const adminAddress = await models.Address.findOne({
+    const address = await models.Address.findOne({
       where: {
         address: req.body.address,
         user_id: req.user.id,
       },
     });
-    if (!adminAddress) return next(new Error('Must be an admin/mod to invite new members'));
+
+    if (!address) return next(new Error(Errors.AddressNotFound));
     const requesterIsAdminOrMod = await models.Role.findAll({
       where: {
         ...chainOrCommObj,
-        address_id: adminAddress.id,
+        address_id: address.id,
         permission: ['admin', 'moderator']
       },
     });
-    if (!requesterIsAdminOrMod) return next(new Error('Must be an admin/mod to invite new members'));
+    if (requesterIsAdminOrMod.length === 0) return next(new Error(Errors.MustBeAdminOrMod));
   }
 
   const { invitedEmail } = req.body;
   if (req.body.invitedAddress) {
-    const existingAddress = await models.Address.findOne({ where: {
-      address: req.body.invitedAddress,
-      chain: req.body.invitedAddressChain,
```

Comment:
@zakhap would be good to get your thoughts here and what was intended with this invitedAddressChain as it was not included anywhere in the existing tests you wrote.

Reply:
I don't recall why the invitedAddressChain is included here as it's obviously not related to the check at hand.